Source/Scene/Cesium3DTile.js

```diff
@@ -136,7 +150,10 @@ define([
             var type = url.substring(url.lastIndexOf('.') + 1);
             var contentFactory = Cesium3DTileContentProviderFactory[type];
 
-            if (defined(contentFactory)) {
+            if (type === 'json') {
```

Comment:
This may be outside the scope of this PR, but would `/type/i.test('json')` be better here, or is lowercase guaranteed?  Also, this looks like it is grabbing the type based on the file extension in the url.  That's generally bad practice and we should be using content types.  Though I imagine that's a spec-related decision.